Assets/MixedRealityToolkit/InputSystem/Sources/BaseGenericInputSource.cs

```diff
@@ -31,26 +35,45 @@ public BaseGenericInputSource(string name, InputType[] capabilities, IMixedReali
 
         public string SourceName { get; }
 
+        public InputSourceState InputSourceState { get; }
+
+        public Handedness Handedness { get; }
```

Comment:
This shouldn't be in the base generic class, but instead in the controller interface.